scrapi/base/helpers.py

```diff
@@ -131,13 +131,20 @@ def maybe_group(match):
 
 
 def gather_object_uris(identifiers):
+    '''
+    Gathers object URIs if there are any
+    >>> gather_object_uris(['nopenope', 'doi:10.10.gettables', 'http://dx.doi.org/yep'])
+    [u'http://dx.doi.org/10.10.gettables', u'http://dx.doi.org/yep']
```

Comment:
This will actually fail in python 3 (stupid unicode...)

need to do 
```python
>>> print(gathered[0]
http://dx.doi.org/10.10.gettables
>>> print(gathered[1]
http://dx.doi.org/yep
```
At least until we can drop python 2 support